restapi/src/main/java/io/stargate/web/docsapi/service/DocsApiConfiguration.java

```diff
@@ -1,41 +1,42 @@
 package io.stargate.web.docsapi.service;
 
-import io.stargate.web.docsapi.dao.DocumentDB;
-
 /** The configuration for the document API. */
 public interface DocsApiConfiguration {
 
   /** Default config instance */
   DocsApiConfiguration DEFAULT = new DocsApiConfiguration() {};
 
   /** @return Returns max allowed document page size that user can request. */
-  default int getMaxPageSize() {
-    return DocumentDB.MAX_PAGE_SIZE;
-  }
+  int MAX_PAGE_SIZE = 20;
```

Comment:
why was this moved to static field instead of the function? the idea was that with the function you can override it in tests with a mocked value. This way we can not?

I would also be fine if you move this to the constants class?